suse_migration_services/units/post_mount_system.py

```diff
@@ -67,3 +67,31 @@ def main():
             Command.run(
                 ['udevadm', 'trigger', '--type=devices', '--action=add']
             )
+
+
+def update_env(preserve_info):
+    proxy_env = {}
+    for _, preserve_files in preserve_info.items():
+        if Defaults.get_proxy_path() in preserve_files:
+            with open(Defaults.get_proxy_path(), 'r') as proxy_file:
+                for line in proxy_file.readlines():
+                    if not (line.startswith('#') or line.startswith(os.linesep)):
+                        key_value = line.lower(). \
+                            replace(os.linesep, ''). \
+                            replace('"', ''). \
+                            split('=')
+                        proxy_env.update(dict([key_value]))
```

Comment:
This creates a "non" compatible environment. For example when configuring a proxy then the local settings are exported into the shell for the "SOCKS" and "NO" proxy settings are both available in upper and lower case, i.e.
`NO_PROXY` and `no_proxy` as well as `SOCKS_PROXY` as well as `socks_proxy` both exist. This implementation forces everything to lower case, which is correct for https and http.

Given that the DMS only cares about https, http, and there may be ftp repos for which lower case only is sufficient at the very least we should comment this code to set the proper scope, i.e. that in DMS we only care about the 3 protocols for which lower case is sufficient.